src/instructlab/model/chat.py

```diff
@@ -175,6 +175,12 @@ def is_openai_server_and_serving_model(
     "--temperature",
     cls=clickext.ConfigOption,
 )
+@click.option(
```

Comment:
will you add other flags to customize the `document_store` and `retriever` sections of the `chat.rag` configuration?

Reply:
Yes.